terraform/main.mk

```diff
@@ -11,8 +11,8 @@ TERRAFORM_STATE_DYNAMODB_TABLE ?= tf-state-lock
 TERRAFORM_STATE_BUCKET_NAME ?= $(NAMESPACE)-tf-state
 CHECKOV ?= $(DOCKER) run -v $(ENV_DIR):/tf -i bridgecrew/checkov -d /tf -s
 TFLINT ?= $(DOCKER) run --rm -v $(ENV_DIR):/data -t wata727/tflint
-TERRAFORM ?= $(shell which terraform)
-#TERRAFORM ?= $(DOCKER) run -w /terraform -v $(PWD)/:/terraform -i hashicorp/terraform:0.12.21 init
+TERRAFORM ?= $(DOCKER) run -v $(ENV_DIR):/$(ENV_DIR) -v "$(ENV_DIR)/.terraform":/"$(ENV_DIR)/.terraform" -v "$(INFRA_DIR)":"$(INFRA_DIR)" -v ~/.aws/:/root/.aws:ro -w $(ENV_DIR) -i -t -e AWS_PROFILE=$(AWS_PROFILE)  hashicorp/terraform:light
```

Comment:
Let's also make sure we use `--rm` as otherwise it would create lots of dead containers

Reply:
Any reason why is there `  ` (double space) between `$(AWS_PROFILE)  hashicorp/terraform:light`?

Reply:
Fixed